pkg/fileutils/fileutils_unix.go

```diff
@@ -20,3 +21,18 @@ func GetTotalUsedFds() int {
 	}
 	return -1
 }
+
+// Exist checks whether a file or directory exists at the given path.
+// It uses unix.Access which is a faster operation compared to os.Stat for
```

Comment:
This should name the function it really calls, if it mentions it at all.

Reply:
I’d lean towards not mentioning it in the API documentation here — the job is to check for existence.

That `Access` is faster is an implementation detail — a non-obvious detail, so worth a comment — but that comment can be moved inside the function.